database/migrations/2024_03_24_104639_create_articles_table.php

```diff
@@ -14,18 +14,19 @@ public function up(): void
         Schema::create('articles', function (Blueprint $table) {
             $table->id();
             $table->string('title')->index();
-            $table->string('slug');
+            $table->string('slug')->unique();
             $table->string('image');
             $table->text('content');
             $table->foreignId('category_id')->references('id')->on('article_categories')->onDelete('restrict')->onUpdate('restrict');
-            // $table->foreignId('user_id')->references('id')->on('users')->onDelete('cascade')->onUpdate('cascade');
```

Comment:
I said if there anything i asked to be set null change it to restrict, user was cascade so don't change it